app/page/accountList/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { useContext, useState } from "react"
+import CircleCheck from "react:~assets/circleCheck"
+import Plus from "react:~assets/plus"
+import Wallet from "react:~assets/wallet"
+import { useHashLocation } from "wouter/use-hash-location"
+
+import { Divider } from "~app/component/divider"
+import { PasskeyVerification } from "~app/component/passkeyVerification"
+import { StepBackHeader } from "~app/component/stepBackHeader"
+import { ProviderContext } from "~app/context/provider"
+import { ToastContext } from "~app/context/toastContext"
+import { useAccounts, useAction, useNetwork } from "~app/hook/storage"
+import { Path } from "~app/path"
+import { AccountType } from "~packages/account"
+import { PasskeyAccount } from "~packages/account/PasskeyAccount"
+import { PasskeyOwnerWebAuthn } from "~packages/account/PasskeyAccount/passkeyOwnerWebAuthn"
+import address from "~packages/util/address"
+import number from "~packages/util/number"
+import type { HexString } from "~typing"
+
+export function AccountList() {
+  const [, navigate] = useHashLocation()
+  const { provider } = useContext(ProviderContext)
+  const { setToast } = useContext(ToastContext)
+  const { createAccount, switchAccount } = useAction()
+
+  const network = useNetwork()
+
+  const accounts = useAccounts()
+  const totalBalance = accounts.reduce((b, a) => {
+    return b + number.toBigInt(a.balance)
+  }, 0n)
+
+  const [accountCreating, setAccountCreating] = useState(false)
+  const createNewAccount = async () => {
+    setAccountCreating(true)
+    try {
+      if (!network.accountFactory[AccountType.PasskeyAccount]) {
+        throw new Error("Passkey account factory is not set")
+      }
+      const account = await PasskeyAccount.initWithFactory(provider, {
+        owner: await PasskeyOwnerWebAuthn.register(),
+        salt: number.random(),
+        factoryAddress: network.accountFactory[AccountType.PasskeyAccount]
+      })
+      await createAccount(account, network.id)
+      setToast("Account created!", "success")
+      navigate(Path.Home)
+    } catch (e) {
+      console.error(e)
+    } finally {
+      setAccountCreating(false)
+    }
+  }
+  const selectAccount = async (accountId: string) => {
+    await switchAccount(accountId)
+    navigate(Path.Home)
+  }
+
+  if (accountCreating) {
+    return <PasskeyVerification purpose="identity" />
+  }
+
+  return (
+    <>
+      <StepBackHeader title="Account List" />
+
+      {/* Total Balance */}
+      <section className="py-[16px]">
+        <div className="mb-[8px] text-[16px]">Total Balance</div>
+        <div className="text-[48px]">
+          {number.formatUnitsToFixed(totalBalance, 18, 2)} ETH
+        </div>
+      </section>
+
+      {/* Account List */}
+      <section className="w-[calc(100%+32px)] h-[264px] ml-[-16px] overflow-x-hidden overflow-y-scroll">
```

Comment:
Note:
This setting is important for handling element overflow. Learned a lot!

Reply:
I think a better way could be not to set padding at root element, we can discuss it later.